docs/workforce-management-framework/topics/con-KeycloakVsPassport.adoc

```diff
@@ -0,0 +1,37 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} currently supports two security implementations
```

Comment:
not quite sure what you are referring to . if its modules then there is no module for keycloak . Or do you want random links pointing to core and angularjs?